Source/GostCryptography.Tests/GostCryptography.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net40;net452</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
It is better to use netstandard2.1 instead of netcoreapp3.1.
More better: netstandard2.0

Why you need a dependency on runtime?